tools/editor/editor_settings.cpp

```diff
@@ -569,6 +569,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 
 	set("import/automatic_reimport_on_sources_changed",true);
 
+        set("create_dialog/start_fully_expanded",false);
```

Comment:
Also, I'd prefer not adding a new `create_dialog` category, the editor settings are already pretty hard to read with an important number of sections with only one parameter in them. Maybe we can find an existing section that would be suitable for this setting?